src/sentry/plugins/bases/notify.py

```diff
@@ -129,7 +129,13 @@ def __is_rate_limited(self, group, event):
             limit=10,
         )
 
+    def is_configured(self, project):
```

Comment:
See my comment. :)

I checked every one that we run on getsentry for it, and they all implement this signature, surprisingly. So my guess is that we used to support this historically, then somehow stopped?